frontend/src/old-pages/Clusters/Scheduling.tsx

```diff
@@ -308,7 +281,7 @@ export default function ClusterScheduling() {
                 {
                     id: "state",
                     header: "state",
-                    cell: item => <Status status={(item as any).job_state}/>,
+                    cell: item => <Status status={(item as any).job_state as string}/>,
```

Comment:
suggestion: it's better to avoid these explicit downcasts as they go against the framework, and it's not gonna help us realize if we mess up with types (since you're forcing a cast(. What you can do is to type `item` properly (or the jobs list directly) so that a cast is not needed.